core/commonMain/src/kotlinx/serialization/internal/PluginGeneratedSerialDescriptor.kt

```diff
@@ -116,8 +118,8 @@ internal inline fun <reified SD : SerialDescriptor> SD.equalsImpl(
     return true
 }
 
-@Suppress("NOTHING_TO_INLINE")
-internal inline fun SerialDescriptor.hashCodeImpl(typeParams: Array<SerialDescriptor>): Int {
+@OptIn(ExperimentalSerializationApi::class)
+internal fun SerialDescriptor.hashCodeImpl(typeParams: Array<SerialDescriptor>): Int {
```

Comment:
Are you sure that we do not want to inline this?

Reply:
Yes, it has no benefits, but bloats bytecode size for nothing